node/narwhal/src/bft.rs

```diff
@@ -48,9 +49,9 @@ pub struct BFT<N: Network> {
 
 impl<N: Network> BFT<N> {
     /// Initializes a new instance of the BFT.
-    pub fn new(account: Account<N>, storage: Storage<N>, ledger: Ledger<N>, dev: Option<u16>) -> Result<Self> {
+    pub fn new(account: Account<N>, storage: Storage<N>, ledger: Ledger<N>, ip: SocketAddr) -> Result<Self> {
```

Comment:
Please preserve the `dev: Option<u16>` syntax to be consistent with the library interface. This will be especially problematic in integration upstream and when storage DB files are needed.